backend/src/Equinor.ProjectExecutionPortal.WebApi/ViewModels/Portal/ApiRemovePortalRequest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Equinor.ProjectExecutionPortal.Application.Commands.Portals.RemovePortal;
+using FluentValidation;
+
+namespace Equinor.ProjectExecutionPortal.WebApi.ViewModels.Portal
+{
+    public class ApiRemovePortalRequest
+    {
+        public Guid Id { get; set; }
```

Comment:
Technically you dont need this property. It is sufficient to only send the ID in via the ToCommand(). Since it is a non-nullable GUID the validator is not needed

Reply:
Removed